src/main/java/net/juniper/netconf/Device.java

```diff
@@ -776,21 +772,21 @@ public void commitFull() throws CommitException, IOException, SAXException {
      * text/xml format.
      *
      * @param configFile Path name of file containing configuration,in text/xml format,
-     *  to be loaded. For example,
-     *  " system {
-     *      services {
-     *        ftp;
-     *      }
-     *    }"
-     *  will load 'ftp' under the 'systems services' hierarchy.
-     *  OR
-     *  "&lt;configuration&gt;&lt;system&gt;&lt;services&gt;&lt;ftp/&gt;&lt;
-     *  services/&gt;&lt;/system&gt;&lt;/configuration/&gt;"
-     *  will load 'ftp' under the 'systems services' hierarchy.
+     *                   to be loaded. For example,
+     *                   " system {
```

Comment:
The indents on the example are intentional. Please revert the example indentation.